webclient/src/index.html

```diff
@@ -199,6 +199,13 @@
                 </ul>
               </div>
             </div>
+            <div class="column hide-lg">
```

Comment:
```suggestion
            <div class="hide-lg">
```

One defect:
The tum logo can now be "pushed behind the search bar"
![image](https://user-images.githubusercontent.com/26258709/234997923-cd079b65-1f44-436b-9fee-cf2dc5852c56.png)

Do you think I can keep hide-lg, or should I add a custom breakpoint for this?